src/test/resources/application-test.yml

```
@@ -1,4 +1,9 @@
 spring:
+  neo4j:
+    uri: ${GRAPH_DB_URL}
+    authentication:
+      username: ${GRAPH_DB_USERNAME}
+      password: ${GRAPH_DB_PASSWORD}
   data:
     neo4j:
-      database: neo4j
+      database: neo4j
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)